ReSharper.FSharp/src/FSharp.Psi.Features/src/CodeCleanup/FantomasHost.fs

```diff
@@ -75,19 +71,11 @@ type FantomasHost(solution: ISolution, fantomasFactory: FantomasProcessFactory,
 
     member x.FormatSelection(filePath, range, source, settings, options, newLineText) =
         connect()
-        if runningVersion >= FantomasProtocolConstants.Fantomas5Version then
-            if showSelectionNotification then
-                notifications.CreateNotification(solutionLifetime,
-                    title = "Selection formatting is not supported",
-                    body = "Specified Fantomas version does not support selection formatting.") |> ignore
-                showSelectionNotification <- false
-            raise (NotImplementedException())
-        else
-            let args =
-                RdFantomasFormatSelectionArgs(convertRange range, filePath, source, convertFormatSettings settings,
-                    convertParsingOptions options, newLineText)
-
-            connection.Execute(fun () -> connection.ProtocolModel.FormatSelection.Sync(args, RpcTimeouts.Maximal))
+        let args =
+            RdFantomasFormatSelectionArgs(convertRange range, filePath, source, convertFormatSettings settings,
```

Comment:
Maybe `convertRange` could be named somehow `toRdFcsRange` or `rdFcsRange`? That would perfectly describe what kind of conversion is happening inside it.